rclpy/rclpy/node.py

```diff
@@ -97,16 +91,16 @@ class Node:
 
     def __init__(
         self,
-        node_name: str,
+        node_name,
         *,
-        context: Context = None,
```

Comment:
I thought python 3.5 supported type hints in parameters. Its on the [3.5 version of the `typing` module](https://docs.python.org/3.5/library/typing.html) documentation.

Reply:
+1 It is also my understanding that type hints for parameters are okay in Python 3.5. I think it's just type hints on lone variables that we need to update.

Reply:
I started clearing these out based on test feedback on a Debian Stretch system but I've been gradually restoring them and not running into issues.

It seems highly desirable to make the minimal changes necessary to reduce merge conflicts so I'll be revising this and the launch PR to restore what I can.

Reply:
Looks like the errors I was getting were related to trailing commas in the parameter lists with type hints and not the type hints themselves.

Reply:
5fd15f0 restores the type hints. Apparently trailing commas are the problem. Anyone who wants to examine further can do so by pulling this into a Debian Stretch workspace.